core/jobs/src/main/java/com/cognifide/aet/job/common/comparators/source/CodeFormatter.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * AET
+ *
+ * Copyright (C) 2018 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.job.common.comparators.source;
+
+import com.cognifide.aet.job.common.comparators.source.visitors.ContentVisitor;
+import com.cognifide.aet.job.common.comparators.source.visitors.MarkupVisitor;
+import com.cognifide.aet.job.common.comparators.source.visitors.NodeTraversor;
+import org.apache.commons.lang3.StringUtils;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+class CodeFormatter {
```

Comment:
Thank you @ernestosiak ! I like that this code is moved to separate class.

Reply:
:)